source/linear-algebra/source/02-EV/07.ptx

```diff
@@ -149,22 +209,34 @@ used in this span?
   </li>
   <li>
       <p>
-        The set spans all of <m>\IR^4</m>.
+        The set spans the solution space.
       </p>
     </li>
-<li>
+  <li>
     <p>
-        The set fails to span the solution space.
+        The set is a basis for the solution space. 
     </p>
   </li>
 </ol>
     </p>
 </task>
 </activity>
+<answer>
+    <p>
+        D.
+    </p>
+</answer>
 
 <sage language="octave">
+    <input>
+    row1=[]
+    row2=[]
+    row3=[]
+    rref([row1;row2;row3])
+    </input>
 </sage>
```

Comment:
I think we usually just have a blank sage cell when we don't provide the matrix terms.